.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\nuts_and_bolts",
+    "\\nuts_and_bolts\\templates"
+  ],
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
delete the .vs folder and commit. I'd suggest using VS Code instead of Visual Studio

Reply:
Will change over tonight

Reply:
Done

Reply:
Need to commit the changes for it to be "done" because those files are still in the branch. Also, my phone goes off with every comment, so let's see less comments and more commits.